packages/app-desktop/gui/NoteEditor/NoteBody/WhiteboardEditor/style.scss

```
@@ -0,0 +1,290 @@
+	border: 1px solid $selection-color;
+	border-radius: 10px;
+	background: var(--joplin-background-color);
+	color: $selection-color;
+	cursor: pointer;
+	z-index: 5;
+}
+
+.wb-card-md {
+	flex: 1;
+	overflow: auto;
+	word-break: break-word;
```

Stylelint (17.11.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)